src/compare-page-pull-request/compare-page-pull-request.js

```diff
@@ -0,0 +1,91 @@
+'use strict';
+
+import { h } from 'dom-chef';
+import elementReady from 'element-ready';
+import { getRepoURL } from '../page-detect';
+import 'selector-observer';
+
+const Constants = {
+    COMPARE_DETAILS_SELECTOR: '#branch-compare.content-container'
+};
+
+export default async function init() {
+    await addUrlIfComparing();
```

Comment:
The `observeSelector` callback should run for nodes that already exist in the DOM, so I think you can actually remove this line, did you try that?

Reply:
This is something I totally missed, I can remove the await call because its all done after dom content ready once the `observeSelector` has fired off